mw-webapp/src/component/tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,41 @@
+.wrapper {
+  position: relative;
+}
+
+.target {
+  cursor: pointer;
+
+  &:hover + .tooltip {
+    display: block;
+  }
+}
+
+.tooltip {
+  display: none;
+  position: absolute;
+  overflow: hidden;
+
+  &_top {
+    bottom: 130%;
```

Comment:
why do you count distance from bottom if it calls top? What will happened if your component height is 10vh?

Reply:
1. I count distance from bootom if I calls top, to avoid writing negative values, for example: ``top: -80%``
2. If my component is given a height to ``vh``, then the display height of the component will depend on the height of the screen. In this case, when specifying ``%``, the height depends solely on the parent block

Reply:
1. what the problem with negative numbers?
2.  i mean how it will look? 30% above tall block means the tooltip will appeared somewhere in the sky, right)?

Reply:
1. Ok, I agree
2. Right. The shell wraps around the child and has the boundaries of that children. The tooltip will be positioned 30% above the child’s border. If this is critical, we have a choice of other positions: left, right, bottom

Reply:
OK so if you will have two elements in a row one tall (text column 2000px) and one with small height (image in this text 50px), then tooltipTop will appeared for one of them 600px above (higher than header) and for  another 15px above. Is it your idea?

Reply:
I got it. In this case, it is better to specify absolute values. Or for such a case, make your own different positioning. Then you need to make one default positioning and remove enam to set custom positioning depending on the case.